src/web/error.rs

```diff
@@ -96,29 +94,6 @@ impl Handler for Nope {
                 .into_response(req)
             }
 
-            Nope::NoResults => {
-                let mut params = req.url.as_ref().query_pairs();
-
-                if let Some((_, query)) = params.find(|(key, _)| key == "query") {
```

Comment:
from what I see, since #1521, `NoResults` was actually never construted while where was a `query` in the GET params.